### ISSUE TITLE
Bug in the bar which appears at the bottom

### ISSUE THREAD

**Issue by paavininanda** [systers/PC-Prep-Kit#72]
On pressing the back button again and again in Activity3, the bottom bar keeps on duplicating itself. A screenshot for the same can be seen
![screen shot 2018-02-16 at 3 45 19 pm](https://user-images.githubusercontent.com/24737071/36310881-63eabadc-134f-11e8-888c-0bbd29b55783.png)

**Comment by paavininanda** [comment]
This bug doesn't exit after #92 got merged. But now the bar doesn't show any progress at all. That is no colour changes on completing an level of the activity.

**Comment by Buddhiprabha** [comment]
@paavininanda can you please close this and open a new one for the new bug in that case?

**Comment by lunayach** [comment]
Also to match when I'm drawing the lines, I was never able to do it at the first time. We may want to increase the empty circle region for starting our line.

**Comment by paavininanda** [comment]
@Buddhiprabha I updated the description and the title. Should I still open a separate issue?
